### PR TITLE
Add compat data for SharedWorker (Worker API)

### PR DESCRIPTION
**Pull request by maboa**
[https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker](https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker)

Not sure whether `Constructor name option` here [https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker](https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker) 
is the same as `name option` defined here [https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker/SharedWorker](https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker/SharedWorker)

### CODE REVIEW

File: api/SharedWorker.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            "version_renoved": "7.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "constructor_name": {
```
Comment (Elchi3) [inline api/SharedWorker.json]: I think it is the same as under the constructor. Remove this entry.

File: api/SharedWorker.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "sharedworker_constructor": {
```
Comment (Elchi3) [inline api/SharedWorker.json]: This identifier would work, too, but our convention for other APIs has been "api.SharedWorker.SharedWorker" for the contructors.

```diff
- "sharedworker_constructor": {
+ "SharedWorker": {
```

File: api/SharedWorker.json [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "5",
+            "version_removed": "6.1"
+          },
+          "safari_ios": {
+            "version_added": "5.1",
+            "version_renoved": "7.1"
```
Comment (Elchi3) [inline api/SharedWorker.json]: typo in "removed" here. That's why the build fails.

File: api/SharedWorker.json [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SharedWorker",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/SharedWorker.json]: Both Android flavors should be `true`.

File: api/SharedWorker.json [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "constructor_name": {
+        "__compat": {
+          "description": "Constructor <code>name</code> option",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorker.json]: Should be `true`.

File: api/SharedWorker.json [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+          "description": "<code>SharedWorker()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/SharedWorker.json]: `true` for both Androids.

File: api/SharedWorker.json [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "name_option": {
+          "__compat": {
+            "description": "<code>name</code> option",
+            "support": {
+              "webview_android": {
+                "version_added": null
```
Comment (jpmedley) [inline api/SharedWorker.json]: Should be `true`.

File: api/SharedWorker.json [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SharedWorker/SharedWorker/port",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/SharedWorker.json]: `true` for both Androids.

File: api/SharedWorker.json (hunk 1 above)
Comment (maboa) [reply api/SharedWorker.json]: 👍

File: api/SharedWorker.json (hunk 2 above)
Comment (maboa) [reply api/SharedWorker.json]: 👍

File: api/SharedWorker.json (hunk 3 above)
Comment (maboa) [reply api/SharedWorker.json]: oops :)

File: api/SharedWorker.json (hunk 4 above)
Comment (maboa) [reply api/SharedWorker.json]: 👍

File: api/SharedWorker.json (hunk 5 above)
Comment (maboa) [reply api/SharedWorker.json]: This section has been removed. See above.

File: api/SharedWorker.json (hunk 6 above)
Comment (maboa) [reply api/SharedWorker.json]: 👍

File: api/SharedWorker.json (hunk 7 above)
Comment (maboa) [reply api/SharedWorker.json]: 👍

File: api/SharedWorker.json (hunk 8 above)
Comment (maboa) [reply api/SharedWorker.json]: 👍

**Pull request merged**